frontend/src/components/DailyView.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useState } from "react";
+import Holidays, { HolidaysTypes } from "date-holidays";
+import axios, { AxiosResponse } from 'axios';
+
+import Cell from "./Cell";
+import { MyDate, Note } from "../types/types";
+import { serverURL } from "../config/config";
+
+const days = ["Sun", "Mon", "Tue", "Wed", "Thr", "Fri", "Sat"];
+const fullDayNames = ["Sunday", "Monday", "Tuesday", "Wendesday", "Thursday", "Friday", "Saturday"];
+const monthsNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
```

Comment:
The constant name should be the upper case.

Reply:
Good for readable code. fixed it.